SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalanceEnforcer/SpatialLoadBalanceEnforcerTest.cpp

```diff
@@ -391,3 +395,87 @@ LOADBALANCEENFORCER_TEST(GIVEN_acl_component_removal_WHEN_request_is_queued_THEN
 
 	return true;
 }
+
+LOADBALANCEENFORCER_TEST(GIVEN_component_presence_change_op_WHEN_we_inform_load_balance_enforcer_THEN_queue_authority_request)
```

Comment:
I think this name is the wrong way around - you have an enforcer in some state, you add a presence change, then an acl change is queued.
It's precondition, call, postcondition.

Reply:
it was following the pattern of other test names in the file, I agree, but for the purposes of PR size I don't want to go adjust all test names in a file when PR is already quite big?